controller/node_controller.go

```diff
@@ -459,6 +459,9 @@ func (nc *NodeController) syncNode(key string) (err error) {
 	if err := nc.syncDiskStatus(node, collectedDiskInfo); err != nil {
 		return err
 	}
+	if err := nc.syncDiskSchedule(node); err != nil {
+		return err
+	}
```

Comment:
When a node is deleted and the deletionTimestamp is set, will the function be called? Who will delete the DiskSchedule CRs for the deleted node?